### PR TITLE
Support for Container Security Containers API

### PR DESCRIPTION
**Pull request by alanfung**
Issue: TAPI-289

### CODE REVIEW

File: tenable_io/api/sc_containers.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    def delete(self, repository_name, sha256):
+        """Delete a container image.
+
+        :param repository_name: The registry/path of the image. For example, if you have a repository named unix and an
+            image named alpine, you would type unix/alpine.
+        :param sha256: SHA256 checksum of the container image.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: True if successful.
+        """
+        uri = 'container-security/api/v1/container/%(repository_name)s/manifests/%%(sha256)s' % \
```
Comment (kagutagu) [inline tenable_io/api/sc_containers.py]: maybe this for consistency? 
`uri = 'container-security/api/v1/container/%(repository_name)s/manifests/%(sha256)s'
        self._client.delete(uri, path_params={'repository_name': repository_name, 'sha256': sha256})`

File: tests/integration/api/test_sc_containers.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class TestScContainersApi(BaseTest):
+
+    def test_delete(self, client):
+        with pytest.raises(TenableIOApiException) as e:
+            client.sc_containers_api.delete(u'test_sc_containers', u'test_sc_containers')
+        assert e.value.response.status_code == 404, u'Request cannot return with 404.'
+        assert json.loads(e.value.response.text)[u'status'] == u'digest_not_found'
+
+    def test_list(self, client):
+        containers = client.sc_containers_api.list()
```
Comment (kagutagu) [inline tests/integration/api/test_sc_containers.py:19]: If list is empty, then containers will be None, and the test will fail. Do we plan to handle empty list scenario?

Comment (alanfung) [reply tests/integration/api/test_sc_containers.py:19]: Are you saying that it is returning None for you? I am expecting that it returns an empty list.

Comment (kagutagu) [reply tests/integration/api/test_sc_containers.py:19]: Yes. It returns None for me.

File: tenable_io/api/sc_containers.py (hunk 1 above)
Comment (alanfung) [reply tenable_io/api/sc_containers.py]: This is actually correct, but I meant to leave a comment to clarify this, which I've forgotten, so thanks for pointing it out.

The reason it's done like this is that the service strangely does not expect `responsitory_name` with slashes `/` properly encoded, which the `delete` method would do. Therefore, I needed to construct that part of the URI first prior to passing it to the `delete` method.

**Review by kagutagu — APPROVED** [review]

**Pull request merged**